packages/node-app/src/tests/giveaway.controller.test.ts

```diff
@@ -193,8 +200,8 @@ describe('POST /giveaways', () => {
   );
 
   it(
-    'should not create new giveaway while not authenticated',
-    notAuthenticated(async () => {
+    'should not create new giveaway while authenticated',
```

Comment:
To make it clearer why it should not create the giveaway even though the user is authenticated:
```suggestion
    'should not create new giveaway if user is not admin',
```